buildtest/cli/buildspec.py

```diff
@@ -1123,6 +1124,14 @@ def summarize_buildspec_cache(configuration):
     [yellow]Total Unique Tags:[/yellow]              [cyan]{len(cache.get_unique_tags())}[/cyan] 
     [yellow]Total Maintainers:[/yellow]              [cyan]{len(cache.get_maintainers())}[/cyan] 
 """
+    if args.pager:
+        with console.pager():
+            summery_print(msg, cache)
+            return
+    summery_print(msg, cache)
+
+
+def summery_print(msg, cache):
```

Comment:
can you add docstring with the two arguments. Just a nitpick but good to document all methods